apps/dashboard/src/components/subscribers/schema.ts

```diff
@@ -27,3 +27,19 @@ export const SubscriberFormSchema = z.object({
     })
     .optional(),
 });
+
+export const CreateSubscriberFormSchema = SubscriberFormSchema.extend({
+  subscriberId: z.string().transform((str, ctc) => {
+    if (!str.trim()) {
+      ctc.addIssue({ code: 'custom', message: 'SubscriberId is required' });
+      return z.NEVER;
+    }
+    return str;
+  }),
+  email: z
```

Comment:
@BiswaViraj if not mistaken we already have a phone validation library as part of the phone input. Let's add a TODO for a following PR to try to validate phones, locales and timezones as well.

Reply:
Yes, currently it validates the phone number as well using the library you mentioned, as the schema is extended from the Edit subscriber schema,
although we do need to validate locales and timezones as it accepts any string as of now